server/src/map/map.helper.js

```diff
@@ -0,0 +1,116 @@
+const map = require('./map.model');
+
+/**
+ * Get all active maps
+ * @return {maps} - return all maps
+ * @return {err} - return error
+ */
+let getAllMaps = () => {
+  return map
+    .find({})
+    .sort({label: 1})
+    .exec()
+    .then((maps) => Promise.resolve(maps))
```

Comment:
just FYI not necesary to resolve it one more time. regardin reject it can be in your case, but better just avoid to add catch to be able handle it one level higher